CleTimer/CleTimer/default/model_test.py

```diff
@@ -0,0 +1,63 @@
+import unittest
+import random
+import model
+
+class ModelTest(unittest.TestCase):
```

Comment:
in the future, I suggest you to use pytest instead of unittest. I find it to be much easier to use. Don't have to change it here

Reply:
:+1: